middleware-jackson/src/main/java/com/nytimes/android/external/store/middleware/jackson/JacksonStringParser.java

```diff
@@ -29,10 +30,10 @@ public JacksonStringParser(@Nonnull ObjectMapper objectMapper, @Nonnull Type typ
     }
 
     @Override
-    @Nullable
-    public Parsed call(@Nonnull String source) {
+    @SuppressWarnings({"PMD.EmptyCatchBlock"})
```

Comment:
Ok, I will fix it.
How about to use [ParserException](https://github.com/NYTimes/Store/blob/develop/store/src/main/java/com/nytimes/android/external/store/util/ParserException.java) instead of generic Exception, to have more specific error?